imports/api/questions.js

```diff
@@ -195,84 +197,56 @@ if (Meteor.isServer) {
     } else this.ready()
   })
 
-  // question library for a user (student can see those they own or created)
-  Meteor.publish('questions.library', function () {
+  Meteor.publish('questions.library', function (courseId) {
```

Comment:
For backwards compatability, should be handle the case where courseId is not specified (e.g. null as default value?)